test/_data/dart_imports_go_first/good.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';  // OK
+import 'dummy.dart';
```

Comment:
Ah, OK.  I see why you want an integration test so that you can define `dummy.dart`.

An alternative would be to simplify things and just ignore the resolution error?

Unless you're really invested in making this an integration test, having tests in the `test/rules` directory is a bit simpler.

I'm not hugely invested either way though! 👍